### PR TITLE
Added titus-container-tools injection

### PR DESCRIPTION
**Pull request by solarkennedy**
I want to make it easier to provide tools to our users.
Specifically, debug tools and perf tools.

For now, I think I want to provide that via
injecting scripts. At first these scripts will
be very simple (scripts that apt-get stuff), but
later the tools may be more sofisticated (statically linked things).

For this PR, I just set up the path. An internal
netflix package will provide the rest.

### CODE REVIEW

**Comment by sargun** [comment]
Did you mean to add  BIN +19.1 MB filesystems/src-logs-010778665/stderr 
?

**Comment by solarkennedy** [comment]
> Did you mean to add BIN +19.1 MB filesystems/src-logs-010778665/stderr
> ?

Lol no

**Comment by solarkennedy** [comment]
I'm going to try different approach with a Volume container instead of bind-mount from the host.

File: executor/runtime/docker/docker_env.go [hunk 1]
```diff
@@ -31,6 +33,11 @@ export {{ $key }}={{ escapeSQWithFallback $key $val }}
 # export {{ $key }}={{ $val | escape_sq }}
 {{ end -}}
 {{ end -}}
+
+# This path setting makes it convenient for users to run
+# tools that have been injected into the container from the
+# Titus agent.
+export PATH=$PATH:/titus/container-tools/bin
```
Comment (rgulewich) [inline executor/runtime/docker/docker_env.go:40]: Do we think our users would be ok with this?  I'm wondering if it's possible to only add this to the path if we're in an ssh session.  Maybe I'm being overly paranoid here. :)

**Review by rgulewich — APPROVED** [review]

File: executor/runtime/docker/docker_env.go (hunk 1 above)
Comment (solarkennedy) [reply executor/runtime/docker/docker_env.go:40]: It is last in the path, should be ok.

**Pull request merged**